src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,77 @@
+package view;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import domain.Ranking;
+import dto.LottoResultDto;
+import dto.LottoTicketDto;
+
+public class OutputView {
+
+	private static final String PURCHASE_MESSAGE = "개를 구매했습니다.";
+	private static final String LOTTO_RESULT_MESSAGE = "당첨 통계\n---------";
+
+	public static void printLottoTicket(LottoTicketDto lottoTicketDto) {
+		lottoTicketDto.getLottoNumbersDto()
+			.stream()
+			.forEach(System.out::println);
+	}
+
+	public static void printLottoPurchaseCount(int purchaseCount) {
+		StringBuilder purchaseCountString = new StringBuilder()
+			.append(purchaseCount)
+			.append(PURCHASE_MESSAGE);
+		System.out.println(purchaseCountString);
+	}
+
+	public static void printLottoResult(LottoResultDto lottoResultDto) {
+		System.out.println(LOTTO_RESULT_MESSAGE);
+		Ranking[] rankingValues = Ranking.values();
+		List<Ranking> rankings = Arrays.stream(rankingValues)
```

Comment:
배열을 지양하고 리스트를 쓰도록 하는게 좋은데요. 이 코드에선 굳이 두 줄로 나눌 필요는 없을 것 같아요!
```suggestion
		List<Ranking> rankings = Arrays.stream(Ranking.values())
```

Reply:
리팩터링 완료했습니다!